src/Code/yield/Worker/TaskManager.js

```diff
@@ -0,0 +1,53 @@
+const versioner = require("../Services/Versioner");
+const { ExternalZenatonError } = require("../../../Errors");
+
+let instance;
+
+const TaskManager = class {
```

Comment:
Strange way to define a class. You can just do `class TaskManager` or I miss something.